wagtailio/static/sass/components/_sub-nav.scss

```diff
@@ -0,0 +1,214 @@
+.sub-nav {
+    @include z-index(nav);
+    $root: &;
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    width: 100%;
+    height: calc(100% - #{$header-mobile-height});
+    position: absolute;
+    list-style: none;
+    top: 0;
+    left: 0;
+    padding: 0 20px;
+    background: $color--white;
+    border-radius: $rounded-xl;
+    // Extra border for high-contrast mode users.
+    border: 1px solid transparent;
+    opacity: 0;
+    visibility: hidden;
+    overflow-y: scroll;
+    overflow-x: hidden;
+    // visible to invisible
+    transition: visibility 0ms ease-out 250ms, opacity 250ms ease-out 0ms;
+    box-shadow: 0 0 30px rgba($color--black, 0.05) ;
+
+    @include media-query(medium) {
+        padding: 0 50px;
+        height: calc(100% - #{$header-tablet-height});
+    }
+
+    @media only screen and (min-width: $header-desktop) {
+        display: none;
+        align-items: flex-start;
+        width: 100%;
+        height: auto;
+        top: 120px;
+        background-color: $color--white;
+        padding: 0;
+        opacity: 1;
+        overflow: hidden;
+        visibility: visible;
```

Comment:
Question: are all the rules under `display: none` needed if the element is hidden?

Reply:
It's togglable, so these are applied when `.active` is